content/docs/guides/data-package-csvw.md

```diff
@@ -0,0 +1,191 @@
+---
+title: How does Data Package compare with CSVW
+sidebar:
+  order: 3
+---
+
+<table>
+  <tr>
+    <th>Authors</th>
+    <td>Peter Desmet, Evgeny Karev, Sara Petti</td>
+  </tr>
+</table>
+
+In 2016, a working group chartered by W3C created a set of recommendations for documenting and accessing CSV files, collectively called [CSVW on the Web (CSVW)](https://www.w3.org/TR/2016/NOTE-tabular-data-primer-20160225/). These show extensive similarities with the Data Package standard. Below we give an overview of the (dis)similarities between the two.
+
+## Scope
+
+Data Package is a set of specifications to describe datasets, data files and tabular data. It can describe data files that are not tabular and includes extra specifications for those that are (i.e. [Table Dialect](/standard/table-dialect/) and [Table Schema](/standard/table-schema/)).
+
+CSVW is a set of specifications to describe delimited text files (CSV, TSV) only.
+
+## Maintenance
+
+Data Package was published in 2012, updated to v1.0 in 2017 and v2.0 in 2024. It is maintained by a [working group](/overview/governance/). Requests for changes can be proposed by anyone as a [GitHub issue](https://github.com/frictionlessdata/datapackage) or pull request and is [decided upon](/overview/governance/#decision-making) by the working group.
+
+CSVW was published in 2016. Its working group was closed at the time. There is currently a community group, but it does [not have the mandate](https://github.com/w3c/csvw?tab=readme-ov-file#the-present) to make changes to the standard.
+
+## Adoption and Software Support
+
+TODO
+
+## Building on Other Standards
+
+Data Package defines its own properties for often-used properties (such as description, contributors, licences, sources), none of which are required. Users can include properties from other standards as [custom properties](/standard/glossary/#custom-properties). It is also possible to add or extend properties by referencing a JSON schema in [@schema](/standard/data-package/#dollar-schema), which are then automatically picked up by validation software.
+
+CSVW makes extensive use of other standards (such as [JSON-LD](https://www.w3.org/TR/json-ld/), [XML Schema Datatypes](https://www.w3.org/TR/xmlschema11-2/), Compact URIs) and defines how to use those.
```

Comment:
In particular, CSVW includes rules how to [connect tabular data with metadata on the Web](https://www.w3.org/TR/tabular-data-model/#locating-metadata) and how to transform data and metadata to RDF.

Reply:
I've added this info as a separate section Linking Data with Metadata